components/dynamic/DataList/styles.js

```diff
@@ -0,0 +1,12 @@
+import styled from "styled-components";
+import { fluidScale } from "@/styles/globalStyles";
+
+export const Header = styled.h2`
```

Comment:
Moved inline styled component stuff into dedicated `styles.js` files wherever was fast and easy to do so in this refactor